ckanext/unaids/assets/react_components/.gitignore

```diff
@@ -0,0 +1,3 @@
+# add a "yarn build" to your ckan entrypoint.py
+# to compile js binaries to this dir at runtime
+*.js
```

Comment:
1. I think it'd be better to keep this ignore statement in top level .gitignore
2. How about naming this directory `build` instead of `react_components`? `build` is a standard for directory holding artifacts (everybody knows it's temporary). Also with this you could ignore the whole directory in .gitignore

Reply:
oh, I wasn't aware `build` is known for being a temp directory. That's a much better idea 👍